src/contrib/testkits/Akka.TestKit.Xunit2/TestKit.cs

```diff
@@ -142,8 +142,12 @@ protected void InitializeLogger(ActorSystem system)
             {
                 var extSystem = (ExtendedActorSystem)system;
                 var logger = extSystem.SystemActorOf(Props.Create(() => new TestOutputLogger(Output)), "log-test");
-                logger.Ask<LoggerInitialized>(new InitializeLogger(system.EventStream), TestKitSettings.TestKitStartupTimeout)
-                    .ConfigureAwait(false).GetAwaiter().GetResult();
+                // Start the logger initialization task but don't wait for it yet
```

Comment:
The idea behind this was to be able to wrap it around the `WaitUntilTestActorReady` call but that's a bit more work than I thought